lib/src/components/NxTag/types.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {ReactNode, ReactElement, ValidationMap, HTMLAttributes} from 'react';
+import * as PropTypes from 'prop-types';
+
+export const NX_TAG_COLORS =
+    ['default', 'light-blue', 'purple', 'pink', 'blue', 'red', 'green', 'orange', 'yellow', 'lime'] as const;
```

Comment:
Up until now we have said that we did not want "default" to be a specifiable color, but rather that it would be what you get when you don't specify a color at all.  But I see you added "default" to this list; has that changed?

Reply:
> Up until now we have said that we did not want "default" to be a specifiable color, but rather that it would be what you get when you don't specify a color at all. But I see you added "default" to this list; has that changed?

No, this was a mistake, reverted here: https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/ab9f115cfb260d3f24164a6ba762114d5aed0133

Reply:
the default tag color would be specified as indigo unless the user decides to pick a different color